app/assets/stylesheets/application.css.scss

```diff
@@ -84,6 +88,18 @@ $font-size: 17px;
   padding: 15px 30px;
 }
 
+// Global links
+.container {
```

Comment:
Ah, I see what you mean by the global now! I think we can just strip this and scope changes to just the navbar, since that's where the problem is. I'll handle that on my end.